src/services/invoiceService.tsx

```diff
@@ -121,10 +121,20 @@ const InvoiceDocument: React.FC<{ data: InvoiceData }> = ({ data }) => (
 							<Image style={styles.productImage} src={product.image} />
 						</View>
 						<Text style={styles.tableCell}>{product.name}</Text>
-						<Text style={styles.tableCell}>{product.price.toFixed(2)} Rwf</Text>
+						<Text style={styles.tableCell}>
+							{(
+								product.price -
+								(product.price * product.discount) / 100
+							).toFixed(2)}{" "}
+							Rwf
+						</Text>
 						<Text style={styles.tableCell}>{product.quantity}</Text>
 						<Text style={styles.tableCell}>
-							{(product.price * product.quantity).toFixed(2)} Rwf
+							{(
+								(product.price - (product.price * product.discount) / 100) *
+								product.quantity
+							).toFixed(2)}{" "}
+							Rwf
 						</Text>
 					</View>
```

Comment:
yes i can see the discounnt is being applied on each product bought but why on total has no discount included  
```suggestion
					
![Screenshot 2024-07-29 133025](https://github.com/user-attachments/assets/d07e0df5-e903-4423-beaa-f80b33cb5cad)
![Screenshot 2024-07-29 133038](https://github.com/user-attachments/assets/2ea110a1-ad30-470b-8aea-6190de01d816)

```

Reply:
I have the same issue too

Reply:
i see that issue too